mnefun/_mnefun.py

```diff
@@ -1129,7 +1140,7 @@ def run_sss_locally(p, subjects, run_indices):
             print('      Processing %s ...' % op.basename(r))
 
             # estimate head position for movement compensation
-            head_pos, annot = _head_pos_annot(p, r, prefix='        ')
+            head_pos, annot, _ = _head_pos_annot(p, r, prefix='        ')
```

Comment:
I don't think we should end up here if `p.movecomp = None`

> Traceback (most recent call last):
>   File ".../nbwr/mnefun_/process.py", line 114, in <module>
>     print_status=True
>   File ".../mnefun/_mnefun.py", line 638, in do_processing
>     outs[ii] = func(p, subjects, run_indices)
>   File ".../mnefun/_mnefun.py", line 879, in run_sss
>     run_sss_locally(p, subjects, run_indices)
>   File ".../mnefun/_mnefun.py", line 1143, in run_sss_locally
>     head_pos, annot, _ = _head_pos_annot(p, r, prefix='        ')
> TypeError: 'NoneType' object is not iterable

Reply:
Indeed, fixed